cli/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "local-home-testing-cli",
+  "version": "0.0.1",
+  "description": "A command-line interface for interacting with Local Home SDK stubs.",
```

Comment:
is that actually using the stubs of the new radio interfaces?

Reply:
The `local-home-testing-cli` project imports the stubs and true radio functionality from `local-home-testing`, yes.